packages/core/src/lib/connection_manager.ts

```diff
@@ -393,12 +412,14 @@ export class ConnectionManager
             )
           );
         }
+        this.toggleOnline();
       })();
     },
-    "peer:disconnect": () => {
-      return (evt: CustomEvent<PeerId>): void => {
+    "peer:disconnect": (evt: CustomEvent<PeerId>): void => {
```

Comment:
When I tested locally, this event handler was not being called at all until I changed it to use the same pattern as the two above